sonar-python-plugin/src/main/java/org/sonar/plugins/python/IpynbNotebookParser.java

```diff
@@ -113,7 +114,7 @@ private boolean parseSourceMultilineString(JsonParser jParser, JsonToken jsonTok
     for (String line : sourceLine.lines().toList()) {
       var countEscapedChar = countEscapeCharacters(line, new LinkedHashMap<>(), tokenLocation.getColumnNr());
       var currentCount = countEscapedChar.get(-1);
-      addLineToSource(line, new IPythonLocation(tokenLocation.getLineNr(),
+      addLineToSource(line + "\n", new IPythonLocation(tokenLocation.getLineNr(),
```

Comment:
Maybe here it would be cleaner to have: 
```suggestion
      addLineToSource(line + "\n", new IPythonLocation(tokenLocation.getLineNr(), ...)
      aggregatedSource.append("\n");
```
For consistency.